### PR TITLE
add spanish translations

### PR DESCRIPTION
**Pull request by CristianCucunuba**
Signed-off-by: cristiancc <cristiancamilo.cucunuba@gmail.com>

### Issue

i18n file for Spanish is missing

### Solution 

Add i18n file for Spanish

### CODE REVIEW

File: src/i18n/es.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      "navigationItem": {
+        "displayName": "Nombre para mostrar",
+        "url": "URL",
+        "isUrlRelative": "Esta URL es relativa",
+        "shouldOpenInNewWindow": "Abrir en una pestaña nueva",
+        "classNames": "Nombres de clases CSS",
+        "isVisible": "Mostrar en el tienda",
+        "isVisibleHelpText": "Haga que este elemento de navegación esté disponible en el tienda tanto para clientes como para administradores.",
+        "isPrivate": "Acceso solo para administrador",
+        "isPrivateHelpText": "Mostrar solo a los administradores. Se utiliza para organizar cambios en la navegación sin exponerla a los clientes. \"Mostrar en la tienda\" debe revisarse.",
```
Comment (Manizuca) [inline src/i18n/es.json]: In this case, i think something like `\"Mostrar en la tienda\" debe estar seleccionado` would be a better option.

**Review by Manizuca — COMMENTED** [review]
Looks ok for me, just a minor comment

File: src/i18n/es.json (hunk 1 above)
Comment (CristianCucunuba) [reply src/i18n/es.json]: Yes, I'm gonna correct it, thank you

**Review by Akarshit — APPROVED** [review]

**Pull request merged**